code/modules/uplink/uplink_items_fulp.dm

```diff
@@ -30,7 +30,56 @@
 	desc = "A handy firing pin that can only be installed into mech weapons. \
 			It also hides the equipped weapon from plain sight. \
 			Only one can fit on a mecha. \
-			This one comes complete with a handy firing pin that can only be installed into mech weapons"
+			This one comes complete with a handy firing pin that can only be installed into mech weapons."
 	item = /obj/item/storage/box/syndicate/bundle_mech
 	cost = 7 //So you cannot use it to get 3 unlocked mech weapons.
-	restricted_roles = list("Roboticist", "Research Director")
\ No newline at end of file
+	restricted_roles = list("Roboticist", "Research Director")
+
+/datum/uplink_item/role_restricted/sith_starter_kit
+	name = "Sith Starter Kit"
+	desc = "Has everything you need to get started with the Dark Side! \
+			Includes genuine Sith sacred texts describing the secrets of saber fighting, \
+			force lightning, force push and a snazzy red light saber."
+	item = /obj/item/storage/box/syndicate/bundle_sith
+	cost = 20
+	restricted_roles = list("Chaplain", "Curator")
+
+/datum/uplink_item/role_restricted/laser_tag_kit_red
```

Comment:
Suggestion: Make this a purchasable cargo crate instead of a traitor item. You'd want to nerf the extreme laser tag vests a bit if you do that, of course, but "toys that can be used as lethal weapons" sounds really on-theme for Cargo.

Reply:
No; Cargo shouldn't have access to this. It's meant to be a semi-stealthy traitor item.